### PR TITLE
Update builtins

### PR DESCRIPTION
**Pull request by mmsc2**

### CODE REVIEW

File: src/vm/runners/builtin_runner/bitwise.rs [hunk 1]
```diff
@@ -201,6 +204,7 @@ impl BitwiseBuiltinRunner {
     }
 
     pub fn get_used_instances(&self, vm: &VirtualMachine) -> Result<usize, MemoryError> {
+        //println!("entre bitwise");
```
Comment (Oppen) [inline src/vm/runners/builtin_runner/bitwise.rs:207]: Delete this.

File: src/vm/runners/builtin_runner/range_check.rs [hunk 2]
```diff
@@ -175,6 +175,7 @@ impl RangeCheckBuiltinRunner {
     }
 
     pub fn get_used_instances(&self, vm: &VirtualMachine) -> Result<usize, MemoryError> {
+        println!("entre range check");
```
Comment (Oppen) [inline src/vm/runners/builtin_runner/range_check.rs:178]: Delete this.

**Review by Oppen — COMMENTED** [review]
LGTM except for the leftover println debugging

**Review by Jrigada — APPROVED** [review]
LGTM remove the print statements and that is all

**Review by SantiagoPittella — APPROVED** [review]

**Review by fmoletta — APPROVED** [review]

**Pull request merged**